libraries/net/node.cpp

```diff
@@ -1003,7 +1041,6 @@ namespace graphene { namespace net {
 
     void node_impl::fetch_sync_items_loop()
     {
-      VERIFY_CORRECT_THREAD();
```

Comment:
What is the rationale for removing this check? It seems like removing it to work properly would indicate a potential problem in the existing code.

Reply:
Ok, looks like this was moved to `activity_tracer`